Feat: Storybook Updates (#271)
vtex-sites/nextjs.store: src/components/ui/Badge/DiscountBadge.stories.mdx

import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs'

import { DiscountBadge } from '.'

import {
  TokenTable,
  TokenRow,
  TokenDivider,
} from 'src/../.storybook/components'

<Meta title="Molecules/Badge/DiscountBadge" component={DiscountBadge} />

export const Template = (args) => <DiscountBadge {...args} />

<header>

# Discount Badge

> A `Badge` used to show product discounts. You can see its usage on [ProductCard](?path=/docs/molecules-productcard--overview-default).

Reviewer: ```suggestion
A custom `Badge` that display product's discounts. You can see its usage on [ProductCard](?path=/docs/molecules-productcard--overview-default).
```

</header>

## Overview

The `DiscountBadge` component uses [Badge](https://www.faststore.dev/reference/ui/atoms/Badge) as base.

<Canvas>
  <Story
    name="overview-default"
    args={{
      big: false,
      listPrice: 45,
      spotPrice: 40,
    }}
  >
    {Template.bind({})}
  </Story>
</Canvas>

> 

Reviewer: Maybe we can also add the other variants here (medium and high discount) 

We've been listing the other variants in the overview section whenever possible. WDYT?

<img width="655" alt="image" src="https://user-images.githubusercontent.com/3356699/196681775-51cb19d6-dd9f-422c-8364-a36231328883.png">
